### PR TITLE
Move the skeleton to the bleeding edge

### PR DESCRIPTION
**Pull request by bartbroere**
As mentioned in @remco-nfi 's release notes, these changes need to be made to support the latest SDK version (0.2.0).

### CODE REVIEW

**Comment by remco-nfi** [comment]
Thanks for the PR!
Can you please update the SDK version to `0.3.0` in setup.py?

File: plugin/extraction_plugin.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
-from hansken_extraction_plugin.api.author import Author
 from hansken_extraction_plugin.api.extraction_plugin import ExtractionPlugin
-from hansken_extraction_plugin.api.maturity_level import MaturityLevel
-from hansken_extraction_plugin.api.plugin_info import PluginInfo
+from hansken_extraction_plugin.api.plugin_info import Author, PluginInfo, MaturityLevel
```
Comment (remco-nfi) [inline plugin/extraction_plugin.py]:
```suggestion
from hansken_extraction_plugin.api.plugin_info import Author, MaturityLevel, PluginInfo
```

**Review by remco-nfi — CHANGES_REQUESTED** [review]

**Comment by bartbroere** [comment]
> Thanks for the PR!
> Can you please update the SDK version to `0.3.0` in setup.py?

Yes, done!

**Comment by bartbroere** [comment]
I think we're done. Feel free to merge it if it looks okay like this.

**Comment by remco-nfi** [comment]
Looks great! Thanks!

**Pull request merged**